Python/bltinmodule.c

```diff
@@ -243,38 +243,41 @@ PyDoc_STRVAR(build_class_doc,
 \n\
 Internal helper function used by the class statement.");
 
+/*[clinic input]
+__import__ as builtin___import__
+
+    name: object
+    globals: object(c_default="NULL") = None
```

Comment:
Is `globals=None` always the same as default? If yes, you can omit `c_default="NULL"`, but looking at the code I see some subtle differences.

Reply:
It should be `NULL` if it is omitted so it is correct.